main.py

```diff
@@ -46,11 +57,20 @@ def createTodaysGames(games, df):
 
 
 def main():
+    odds = None
     data = get_todays_games_json(todays_games_url)
     games = create_todays_games(data)
     data = get_json_data(data_url)
     df = to_data_frame(data)
-    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, df)
+    if args.odds:
+        print("--------------Automatic odds data scraping-------------")
```

Comment:
I think we should remove this print statement. One thing thats not needed but maybe a nice to have would be a pretty print of the odds for the games.

Reply:
done